src/main/cpp/t07_ip.cpp

```diff
@@ -34,5 +34,36 @@
 using namespace std;
 
 int t07_ip() {
-
+short dots = 0;
+string ip = "", currnum = "";
+getline(cin, ip);
+for (int i = 0; i <= ip.length(); i++)
+{
+    if (ip[i] != '.' && i != ip.length())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [When i==ip.size(), ip[i] is out of bounds.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594756)